README.md

```diff
@@ -179,6 +187,23 @@ $.Function :: Type
 
 Type comprising every Function value.
 
+#### `NonZeroFiniteNumber`
+
+```haskell
+$.NonZeroFiniteNumber :: Type
+```
+
+Type comprising every Number value except `Infinity`, `-Infinity`, `NaN`, `0` 
+and `-0`. 
```

Comment:
I'd prefer to describe this type as comprising every `FiniteNumber` value except `0` and `-0` (and their object counterparts).